### ISSUE TITLE
trouble in tracing white roots

### ISSUE THREAD

**Issue by Abieskawa** [SmartRoot/SmartRoot#15]
Does this plugin only work on those black roots in white background? I cannot trace the grey roots' length in black background !

**Comment by guillaumelobet** [comment]
Yes, it does. You just need to invert the image in the ImageJ menu ("Edit > Invert")
